### DISCUSSION TITLE
How to make changes to the project in our local machine?

### DISCUSSION THREAD

**Discussion by yaten2302** [anuraghazra/github-readme-stats#3139] [Q&A] [answered]
#### Describe the Problem
Hi, to make changes in this project in our own computer, do we have to add some more details except ```PAT_1``` in the ```.env``` file?
As, whenever I'm running the ```vercel dev``` command(after making changes in the code), I'm not able to see the changes. <br>
![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/1fb87692-c68d-4472-9ad3-72c4a74ca804) <br><br>
And also, do we have to remove all these from the .gitignore file?<br>
![image](https://github.com/anuraghazra/github-readme-stats/assets/129659514/16426d03-4674-47f4-84ca-66270ccdd595)

**Comment by rickstaa** [comment, marked as answer]
@yaten2302 thanks for your question. There is not need to set the `PAT_1` manually since they should be pulled from the Vercel instance when you run `vercel dev` (see https://vercel.com/docs/projects/environment-variables#development-environment-variables). If you have any more questions feel free to ping me.

**Reply by yaten2302** [reply]
Got it👍, thank you so much
